docs/cas-server-documentation/integration/Configuring-Fortress-Integration.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: default
+title: CAS - Fortress Authentication
+---
+
+# Apache Fortress Authentication
+
+Services connected to CAS can also use Apache Fortress to handle the authentication and authorization with fortress. The Idea of this is because Apache Fortress doesn't have any SSO mechanism. However, Apache Fortress is complied for ANSI INCITS 359 RBAC.  
+[Here](http://directory.apache.org/fortress/testimonials.html) is the link how it was worked before i submit this feature for cas.
+
+To enable this support's feature, make sure Apache Fortress have been  [installed](http://directory.apache.org/fortress/installation.html "apache fortress installation") and include this dependency in your war overlay project:  
+```xml
+<dependency>
+    <groupId>org.apereo.cas</groupId>
+    <artifactId>cas-server-support-fortress</artifactId>
+    <version>${cas.version}</version>
+</dependency>
+```  
+
+For this current version, Fortress will only support using tomcat as the web container. However, we will try to support other web container as well in the future.
+
+## Server side configuration
+
+### Configure Apache Fortress Properties
+Configure `fortress.properties` file and put it under your $TOMCAT_HOME/lib or you can append your own classpath configuration.  
+
+### Fortress Realm Proxy
+Put Fortress Realm Proxy under your $TOMCAT_HOME/lib
+
+### Fortress Version
+Add `-Dversion=<your.fortress.version>` to `JAVA_OPTS` or `CATALINA_OPTS.  
+It is also possible if you would like to add it to your overlay spring definition :
+```xml

```

Comment:
You mean in the CAS overlay? I would much rather this turns into Java config where the version is configurable via FortressProperties. Possible to do?